src/main/java/fi/aalto/cs/apluscourses/intellij/actions/GetSubmissionsDashboardAction.java

```diff
@@ -44,47 +33,11 @@ public GetSubmissionsDashboardAction(@NotNull MainViewModelProvider mainViewMode
 
   @Override
   public void actionPerformed(@NotNull AnActionEvent e) {
-    Project project = e.getProject();
-    MainViewModel mainViewModel = mainViewModelProvider.getMainViewModel(project);
-    ExerciseDataSource exerciseDataSource = mainViewModel.getExerciseDataSource();
-
-    if (exerciseDataSource == null) {
-      return;
-    }
-
-    APlusAuthentication authentication =
-        (APlusAuthentication) exerciseDataSource.getAuthentication();
-
-    Course course = requireNonNull(mainViewModel.getCourseViewModel().get()).getModel();
-
-    SubmissionResultsList submissionResultsList = tryGetSubmissionsDashboard(
-        Long.parseLong(course.getId()), authentication, requireNonNull(project));
-    course.setSubmissionsDashboard(requireNonNull(submissionResultsList));
+    /*
+    This action will eventually be a generic update action, which updates the main view model. For
```

Comment:
why not to remove the class completely?

Reply:
Hmm good point, it's still in the git history should we need it again